cypress/integration/01-organization_sign_up_spec.js

```diff
@@ -181,13 +181,6 @@ describe("Organization sign up", () => {
         {
           exclude: [],
```

Comment:
You can also nix this `exclude` portion, and just have the call to `cy.checkA11y()` without any parameters.

Reply:
Ok for real this time I removed this!